python/interpret-core/interpret/glassbox/_ebm/_ebm.py

```diff
@@ -2145,6 +2145,129 @@ def monotonize(self, term, increasing="auto", passthrough=0.0):
 
         return self
 
+    def remove_terms(self, term_list):
```

Comment:
I'd like to call this "terms" instead of "term_list" and accept any enumerable object (could be a numpy array for instance)